firefox-ios/Client/Frontend/PasswordManagement/PasswordManagerViewModel.swift

```diff
@@ -74,16 +74,12 @@ final class PasswordManagerViewModel {
 
     /// Searches SQLite database for logins that match query.
     /// Wraps the SQLiteLogins method to allow us to cancel it from our end.
-    func queryLogins(_ query: String, completion: @escaping (([LoginRecord]) -> Void)) {
+    func queryLogins(_ query: String, completion: @escaping (([EncryptedLogin]) -> Void)) {
```

Comment:
Not mandatory, but I'm wondering if you consider removing the extra pair of parentheses from the `completion` function call? This would help streamline the code a bit. Thank you!!